src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/EnumExtensions.cs

```diff
@@ -30,5 +31,16 @@ private static EnumLabelValueAttribute GetEnumLabelValueAttribute(this Enum enum
             return enumValue.GetEnumAttribute<EnumLabelValueAttribute>();
         }
         
+        
+        public static List<EnumValue> GetValues<T>()
+        {
+            return (from object itemType in Enum.GetValues(typeof(T)) select new EnumValue() {Name = Enum.GetName(typeof(T), itemType), Value = (int) itemType}).ToList();
+        }
+        
+        public class EnumValue
+        {
+            public string Name { get; set; }
```

Comment:
I'm not sure about this. At the very least, T can be constrained - `where T: Enum`
but returned here is the index position of the enum items and if they were to every change (e.g. someone inserts a new one) then it could break whatever uses EnumValue.

Reply:
Created EES-734